tasks/Task.hpp

```diff
@@ -4,10 +4,17 @@
 #define SONAR_BVT_SDK_TASK_TASK_HPP
 
 #include "sonar_bvt_sdk/TaskBase.hpp"
-//#include "bvt_c/bvt_sdk.h"
 #include <stdio.h>
 #include "bvt_sdk.h"
+#include "bvt_cpp/bvt_imageres.hpp"
 #include <iostream>
+#include <stdexcept>
```

Comment:
Move the headers that are not in use in the header in the cpp file (looks like most of them)